include/mathfu/vector.h

```diff
@@ -410,6 +410,25 @@ class Vector {
     MATHFU_VECTOR_OPERATION(vector->data[i] = data_[i]);
   }
 
+  /// @brief Compare 2 Vectors of the same size
+  ///
+  /// @return True if the 2 vectors contains the same value
```

Comment:
Maybe changing the Vector<float, ANYSIZE> template to compare floats using  `v1.LengthSquared()-v2. LengthSquared() < epsilon` and adding a note saying something like:

> Vectors of float type uses the machine epsilon to compare float values

What do you think? I could send a commit with this tomorrow.